contents/docs/hogql/expressions.md

```diff
@@ -0,0 +1,81 @@
+---
+title: HogQL expressions
+sidebar: Docs
+showTitle: true
+availability:
+    free: full
+    selfServe: full
+    enterprise: full
+---
+
+HogQL expressions enable you to directly access, modify, and aggregate data in PostHog using SQL. You can use them nearly everywhere where event filters exist, including:
+
+- Dashboards
+- Data series
+- Breakdowns
+- Funnels
+- Event explorer
+
+![HogQL trends breakdown filter](../../images/features/hogql/trends-breakdown.png)
+
+HogQL expressions can access data like:
+
+- event `properties`
+- person `properties`
+- `event`
+- `elements_chain`
+- `timestamp`
+- `distinct_id`
+- `person_id`
+
+They then use SQL functions to access, filter, modify, or aggregate the data. A full list of SQL functions are found in [supported ClickHouse functions](/docs/hogql/clickhouse-functions) and [supported aggregations](/docs/hogql/aggregations).
+
+> **Tip:** If you're having trouble getting results from you expression, try testing with a different visualization (trends table often works best) or break down your expression into pieces and test each one.
```

Comment:
>  try testing with a different visualization (trends table often works best)

Can you elaborate why?

Reply:
Basically, you can see all the values returned. Added that